package.json

```diff
@@ -130,6 +131,11 @@
     "react-content-loader": "^4.2.2",
     "react-native-shimmer-placeholder": "^1.0.35"
   },
+  "rnpm": {
```

Comment:
We've decided to remove fonts, so there is no reason for this configuration property